frontend/src/metabase/ui/components/theme/ThemeProvider/ThemeProvider.unit.spec.tsx

```diff
@@ -8,7 +8,9 @@ import { Text } from "../..";
 
 import { ThemeProvider } from "./ThemeProvider";
 
-describe("ThemeProvider", () => {
+// TODO: Add substitute tests since we can't test CSS custom properties with JSDom
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip("ThemeProvider", () => {
```

Comment:
This suite failed because of [this change](https://github.com/metabase/metabase/pull/46491/files/fc2e98e8f156180bc6d409d16068e775ff685ae5..e854a269760baf25bd4797138b7344ff00118667#diff-2251b467690450fabf6b0cea69b76558e1d5b9e90116b41611a3b3e5ee5b282eR6).

I'll follow up with this later, but for now, we have a bunch of loki tests setup, so we should be fine.